ci-operator/config/openshift/build-test-images/openshift-build-test-images-master.yaml

```diff
@@ -0,0 +1,50 @@
+base_images:
+  base:
+    name: "4.7"
+    namespace: ocp
+    tag: cli
+  ocp_builder_rhel-8-golang-1.15-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.6
+  ocp_builder_rhel-8-golang-1.15-openshift-4.7:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.7
+  ocp_builder_rhel-8-golang-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-openshift-4.6
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: rhel-8-release-openshift-4.6
+images:
+- context_dir: roots2i
+  dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.15-openshift-4.7:
+      as:
+      - registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7
```

Comment:
i don't think this is doing anything.

Reply:
(it's going to replace `registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7` in the dockerfile, with whatever `ocp_builder_rhel-8-golang-1.15-openshift-4.7` resolved to, but since `registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7 doesn't actually appear in the dockerfile, it's not going to do anything.